server/server.go

```diff
@@ -137,6 +139,10 @@ func NewServer(ctx context.Context, prof Profile, logger *zap.Logger, loggerLeve
 	if err != nil {
 		return nil, fmt.Errorf("failed to init branding: %w", err)
 	}
+	deploymentID, err := s.initDeploymentID(ctx, storeInstance)
```

Comment:
initBranding and initDeploymentID could be put inside initSetting and the following code uses config.deploymentID